native_client/alphabet.h

```diff
@@ -17,29 +18,29 @@ class Alphabet {
   Alphabet(const char *config_file) {
     std::ifstream in(config_file, std::ios::in);
     unsigned int label = 0;
+    space_label_ = -2;
     for (std::string line; std::getline(in, line);) {
       if (line.size() == 2 && line[0] == '\\' && line[1] == '#') {
         line = '#';
       } else if (line[0] == '#') {
         continue;
       }
-      label_to_str_[label] = line;
+      //TODO: we should probably do something more i18n-aware here
+      if (line == " ") {
+        space_label_ = label;
+      }
+      label_to_str_.push_back(line);
       str_to_label_[line] = label;
       ++label;
     }
+    label_to_str_.push_back("*");
```

Comment:
Hm, some context / comment here to help understand ?

Reply:
This is a leftover hack, will fix.